packaging/suse/portusctl/man/markdown/portusctl-setup.md

```diff
@@ -0,0 +1,226 @@
+PORTUSCTL 1 "portusctl User manuals" "SUSE LLC." "AUGUST 2016"
+==============================================================
+
+# NAME
+portusctl setup \- Configure a Portus instance
+
+# SYNOPSIS
+
+**portusctl setup** [options...]
+
+# DESCRIPTION
+The **setup** command allows administrators to configure a Portus instance. Note
+that this command will effectively wipe out any previous configuration. For this
+reason, it's recommended to use this command only when setting up Portus for the
+first time.
+
+# OPTIONS
+**--secure**
+  Toggle SSL usage for Portus. It defaults to true and we really encourage the
+  default behavior.
+
+**--ssl-gen-self-signed-certs**
+  Both Portus and the Docker registry need to have SSL certificates to encrypt all
+  the communications. With this flag, you are telling **portusctl** to
+  automatically generate self-signed certificates. This is useful when testing
+  Portus, but for production cases we recommend using the **ssl-certs-dir** option.
+
+**--ssl-certs-dir**
+  Provide **portusctl** the location of your SSL certificates. These certificates
+  are going to be used to encrypt the communications between the Docker registry
+  and Portus. Note that under the specified directory, **portusctl** will look for
+  a certificate key named *\<hostname\>-ca.key* and a certificate file named
+  *\<hostname\>-ca.crt*.
+
+**--ssl-organization**
+  The organization of the SSL certificate. This only applied if you have used the
+
+**--ssl-organization-unit**
+  The organization unit of the SSL certificate. This only applied if you have used
+  the **ssl-gen-self-signed-certs** option. Defaults to "SUSE Portus example".
+
+**--ssl-email**
+  The email to be used for the SSL certificate. This only applied if you have used
+  the **ssl-gen-self-signed-certs** option. Defaults to "kontact-de@novell.com".
+
+**--ssl-country**
+  The country code for the SSL certificate. This only applied if you have used
+  the **ssl-gen-self-signed-certs** option. Defaults to "DE".
+
+**--ssl-city**
+  The city to be used for the SSL certificate. This only applied if you have used
+  the **ssl-gen-self-signed-certs** option. Defaults to "Nueremberg".
```

Comment:
I guess this doesn't really matter, but it should be either "Nuremberg" or "Nuernberg".